libs/s25main/Loader.cpp

```diff
@@ -907,6 +920,24 @@ void Loader::fillCaches()
                           wineaddon::bobIndex[fat ? wineaddon::BobTypes::FAT_CARRIER_CARRYING_WINE :
                                                     wineaddon::BobTypes::THIN_CARRIER_CARRYING_WINE]
                           + static_cast<unsigned>(imgDir))));
+                    } else if(leatheraddon::isLeatherAddonGoodType(ware))
+                    {
+                        leatheraddon::BobTypes carrierEnum;
```

Comment:
Could this be an extra function (leather ware->leather bob index)? I'd also suggest an assertion as it currently isn't directly clear, that `carrierEnum` will always be initialized

Reply:
done